### PR TITLE
feat(normalization-test): Add a test server for event normalization

### PR DESCRIPTION
**Pull request by tonyo**
Related to https://github.com/getsentry/sentry/pull/10165.
This new management command spawns a unix socket server that accepts unprocessed events, performs normalization, and sends the result back.
We'll need this for test/evaluation purposes.

### CODE REVIEW

File: src/sentry/management/commands/serve_normalize.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    def handle_data(self):
+        result, error = None, None
+        start, end = None, None
+        try:
+            data, meta = self.decode(self.data)
+            start = time.time()
+            result = self.process_event(data, meta)
+            end = time.time()
+        except Exception as e:
+            error = e.message + ' ' + traceback.format_exc()
```
Comment (untitaker) [inline src/sentry/management/commands/serve_normalize.py]: Can't this crash with encoding errors as well, due to unicode coercion?

**Review by untitaker — APPROVED** [review]
I am not sure about the errorhandling but I don't know how much it matters

File: src/sentry/management/commands/serve_normalize.py (hunk 1 above)
Comment (tonyo) [reply src/sentry/management/commands/serve_normalize.py]: I guess you're right, I plugged some `force_str`, just in case.

**Comment by untitaker** [comment]
lgtm

**Pull request merged**